napari/components/viewer_model.py

```diff
@@ -556,10 +557,14 @@ def add_points(
             Symbol to be used for the point markers. Must be one of the
             following: arrow, clobber, cross, diamond, disc, hbar, ring,
             square, star, tailed_arrow, triangle_down, triangle_up, vbar, x.
-        size : float, array
+        size : float, tuple
             Size of the point marker. If given as a scalar, all points are made
-            the same size. If given as an array, size must be the same
-            broadcastable to the same shape as the data.
+            the same size. If given as a tuple, size must be the same length
+            as the number of points.
+        anisotropy : tuple, optional
+            List of anisotropy factors, must be one per dimension. These act as
+            pre-multipliers on the size of each point. If not provided,
```

Comment:
I think it's a bit confusing how the anisotropy factor really only applies to the dimensions out of the viewer plane even though they are defined for all planes. Perhaps we should make a note about that here.

```suggestion
            pre-multipliers on the size of each point. Note that the anisotropy is only displayed for dimensions out of plane for the view. If not provided,
```

Reply:
agreed, added